default/stringtables/en.txt

```diff
@@ -15128,9 +15132,37 @@ Cloak Cross-Interference
 TRANSPATIAL_CLOAK_INTERACTION
 Transpatial Drive - Cloak Interaction
 
+FUEL_TITLE
+Fuel
+
+FUEL_EFFICIENCY_TITLE
+Fuel Efficiency
+
+FUEL_REFUEL_EFFICIENCY_TEXT
+'''[[metertype METER_FUEL]] allows ships to travel the starlanes outside of [[metertype METER_SUPPLY]] of your empire. Fuel is consumed for each starlane traversed outside of supply; stopping within [[metertype METER_SUPPLY]] lines will fully refuel a ship. If a ship remains stationary beyond friendly [[metertype METER_SUPPLY]] lines, fuel will slowly regenerate.
+
+The base fuel capacity of a <encyclopedia ENC_SHIP_DESIGN>Ship Design</encyclopedia> depends on the [[encyclopedia ENC_SHIP_HULL]] used, but it can also be increased using some [[encyclopedia ENC_SHIP_PART]]s.
+
+Ship hulls have a great (400%), good (200%), normal (100%), or bad (60%) efficiency which determines how effective the base fuel capacity can be used. 60% efficiency means for five base fuel you get three jumps. 200% efficiency means you can do two jumps per base fuel capacity. Fuel efficiency typically decreases with hull mass, so only very small ships have a high fuel efficiency and huge ships struggle outside of [[metertype METER_SUPPLY]]. The game always show the effective fuel levels.'''
```

Comment:
always show the effective fuel levels -> will always show the effective fuel levels, after scaling for hull fuel efficiency